### PR TITLE
tpm

### PR DESCRIPTION
**Pull request by allan**

### CODE REVIEW

File: srcpkgs/trousers/files/tcsd/run [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+exec 2>&1
```
Comment (xtraeme) [inline srcpkgs/trousers/files/tcsd/run]: I'd prefer to create `/var/lib/tpm` in the runit script instead.

File: srcpkgs/trousers/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'trousers'
+pkgname=trousers
+version=0.3.13
+revision=1
+build_style=gnu-configure
+conf_files="/etc/tcsd.conf"
+makedepends="libressl-devel"
+system_accounts=tss
```
Comment (xtraeme) [inline srcpkgs/trousers/template]: The homedir for the `tss` user probably needs to be set to `/var/lib/tpm`:

`tss_homedir=/var/lib/tpm`

**Comment by allan** [comment]
adjusted and added some musl patches

File: srcpkgs/trousers/files/tcsd/run [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+chown tss:tss /etc/tcsd.conf
```
Comment (Gottox) [inline srcpkgs/trousers/files/tcsd/run]: ... but changing owners of config files is none of the `run` scripts business. :)

**Comment by Gottox** [comment]
Please fix the above comments, otherwise this PR will be closed.

**Comment by Vaelatern** [comment]
@Gottox, would you mind if I grabbed this, fixed the above issues, and re-PRed from my account? @allan, with your permission of course, since you seem to be no longer interested.

**Comment by Gottox** [comment]
Feel free. I don't care who does the PR. ;)

File: srcpkgs/trousers/template (hunk 2 above)
Comment (Vaelatern) [inline srcpkgs/trousers/template]: This was taken care of in the line below this note.

**Comment by xtraeme** [comment]
Merged in #3334

**Pull request closed without merging**